lxd/storage/backend_lxd.go

```diff
@@ -1840,6 +1840,12 @@ func (b *lxdBackend) imageConversionFiller(imgPath string, imgFormat string) fun
 			return -1, fmt.Errorf("qemu-img convert: failed to convert image from %q to %q format: %v", imgFormat, "raw", err)
 		}
 
+		// Remove the image after the conversion to free up the space as soon as possible.
+		err = os.Remove(imgPath)
```

Comment:
Is the file also removed on error?

Reply:
No, as far as I know if the error occurs the file removal failed.

If the question is whether we should ignore the error:

We can ignore the error, since at this point conversion was successful, so it is a bit of a waste to revert everything just because the temporary file was not successfully deleted.

Or, we can error out to let the user know that there is some file left over. I've never reached this error, but if error occurs we will at least be aware of it (aiming at permission - the file has 0600 permission).

Reply:
I meant if an error occurs earlier than this, does the file still get removed?

Reply:
Ahh, yes. Before the imageConversionFiller is invoked, the file is created in `createInstanceFromConversion` and there is a `defer` that cleans the file up.